go-controller/pkg/ovn/obj_retry.go

```diff
@@ -1140,13 +1161,15 @@ func (oc *Controller) resourceRetry(r *retryObjs, lre *localRetryEntry, now time
 		r.deleteRetryObj(objKey, false)
 		return
 	}
-	entry.backoffSec = entry.backoffSec * 2
-	if entry.backoffSec > 60 {
-		entry.backoffSec = 60
```

Comment:
I see that the exponential increase (and upper limit to 60s) in the backoff timeout has been removed. Now we retry either at every second (`initialBackoff`) or immediately when we force it. Is this also the wanted behavior?

Reply:
I think the original code was wrong actually. The backoff would be doubled before checking if during this iteration if enough had time had passed to retry. This means that setting an initial backoff of 1 second actually would not attempt the first retry until 2 seconds. I think I've fixed this.